### DISCUSSION TITLE
Multispeaker-Finetuning on Single-Speaker-VITS-Model

### DISCUSSION THREAD

**Discussion by jtotheakob** [coqui-ai/TTS#2322] [General Q&A]
Hi all, 

I'm new to the world of TTS and really enjoy playing around with finetuning VITS models. I also could achieve great results with a relatively small dataset. However, I've come to a point, where I'd like to do a multispeaker finetuning. The only pretrained model in german however was trained on a single speaker. In a next step, I added a second voice and thus trained a multispeaker model from the single speaker VITS model. This worked reasonably well, but the result of the voices is worse than when I finetuned the voices seperately. Now here is my question: Is there a possibility to get the same or even a better quality (because of a larger data set of two speakers) by a multispeaker finetuning?
I would be very grateful for your help!

Jakob

**Comment by huongtxvega** [comment]
I have the same question.
